script.js

```diff
@@ -21,4 +22,189 @@ document.querySelectorAll('.nav-item').forEach(n =>
+        link3: 'BootStrap',
+        live: '#',
+        source: '#',
+        img: 'assets/card6.png',
+    },
+];
+
+const postCards = document.querySelector('.post');
+
+postCards.innerHTML += `
+            <img src="${post_details[0].img}"
+             alt="Placeholder"
+             />
+            <div class="multi-post">
+                <h2>${post_details[0].title}</h2>
+                <p>
+                    ${post_details[0].description}
+                </p>
+                <ul>
+                    <li>${post_details[0].link1}</li>
+                    <li>${post_details[0].link2}</li>
+                    <li>${post_details[0].link3}</li>
+                </ul>
+                <button class="btn-orange" id="see-pro" type="button">See Project</button>
+            </div>
+`;            
+
+const card_details = document.querySelector('.project-card');
+
+for (let i = 1; i < post_details.length; i += 1){
+    const cards = `
+            <div class="card ${post_details[i].Image}">
+                <h1>${post_details[i].title}</h1>
+                <p>
+                    ${post_details[i].description}
+                </p>
+                <ul>
+                    <li>${post_details[i].link1}</li>
+                    <li>${post_details[i].link2}</li>
+                    <li>${post_details[i].link3}</li>
+                </ul>
+                <button class="btn-orange card-btn" id="project2" type="button">See Project</button>       
+            </div>
+    `;
+    card_details.innerHTML += cards; 
+}
+
+const popup = document.querySelector('.popup-container');
+
+ popup.innerHTML += `
+        <div class="overlay"></div>
+        <div class="popup">
+            <div class="popup-contain">
+                <div class="row justify-content-space-between">
+                    <h2 class="pop-title"></h2>
+                    <div class="close"><i class="fa-solid fa-xmark"></i></div>
+                </div>
+                <div class="row links">
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>BootStrap</li>
+                    </ul>
+                </div>
+                <div class="popup-content">
+                    <img src="" alt="Placeholder" class="pop-img">
+                    <div class="row column">
+                        <p class="pop-txt"></p>
+                        <div class="btn-div">
+                            <a class="btn-orange pop-btn" id="live" href="">See Live  <i class="fa-solid fa-globe"></i></a>
+                            <a class="btn-orange pop-btn" id="source" href="">See Source <i class="fa-brands fa-github"></i></a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    `;
+
+
+const overlay = document.querySelector('.overlay');
+
+function popupshow() {
+    popup.classList.add('active');
+    overlay.classList.add('active');
+}
+

```

Comment:
- [ ] I  noticed that your popup has just one image for all the projects. That should not be so. The image on the popup should be generated randomly based on the project that was clicked. the image on the project should be the one on the popup. I'll suggest you fix this too. :+1: :smile: